lib/warehouse_messenger/dao/containertypes_dao.pm

```diff
@@ -0,0 +1,85 @@
+package warehouse_messenger::dao::containertypes_dao;
+
```

Comment:
It is a dao for the Clarity containertypes resource. It is needed because the Fluidigm message requires plate size to be sent.